cloudinfo/icdversions_test.go

```diff
@@ -0,0 +1,46 @@
+package cloudinfo
+
+import (
+	"testing"
+
+	"github.com/IBM/cloud-databases-go-sdk/clouddatabasesv5"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetAvailableIcdVersions(t *testing.T) {
+
+	mockType := "icd"
+	mockVersion1 := "1.0.0"
+	mackStable := "stable"
+	mockVersion2 := "2.0.0"
+	mackBeta := "stable"
+
+	// Create a mock ListDeployables method to return a pre-defined response
+	infoSvc := CloudInfoService{
+		authenticator: &core.IamAuthenticator{},
```

Comment:
This should not be done in a unit test, as it will try and connect with the cloud. Usage of an interface for the service itself will allow you to mock up a version of that service, which can then be injected here.